### PR TITLE
Fix #79, add hostname to job report

### LINKED ISSUE #79: Attach hostname in build report

### PR DESCRIPTION
**Pull request by relekang**

### CODE REVIEW

File: frigg_worker/jobs.py [hunk 1]
```diff
@@ -55,6 +55,7 @@ class Job(object):
     pull_request_id = None
     coverage = None
     finished = False
+    worker_host = None
```
Comment (frecar) [inline frigg_worker/jobs.py]: should we maybe say "unknown" or something, so we don't submit None ?

File: tests/test_jobs.py [hunk 2]
```diff
@@ -50,6 +50,7 @@
     'dispatcher_token': 'tokened',
     'hq_url': 'http://example.com/hq',
     'hq_token': 'tokened',
+    'worker_host': 'albus.frigg.io'
```
Comment (frecar) [inline tests/test_jobs.py]: is this used as a integration test, or?

Comment (relekang) [inline tests/test_jobs.py]: It is used on line [71 in this file](https://github.com/frigg/frigg-worker/pull/80/files#diff-ee26cc8025d70ef09f379603aa48a64fR71)

Comment (frecar) [inline tests/test_jobs.py]: ah yes, but the hostname is not "real" ?  it could be dummy.dummy.com ? (not that it should)

Comment (relekang) [inline tests/test_jobs.py]: Ah, yeah its just dummy data :)

**Comment by frecar** [comment]
:+1:

**Pull request merged**